tests/corpus/writer/test_writer.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import json
+import os
+import pytest
+
+
+class TestWriteJson:
```

Comment:
Test classes are there for readability only anyway, so this is subjective. But I prefer to make the test class name correspond exaclty to the tested class, and have only one per tested class. In this case, this would mean:
1. rename this one to `TestWriter`
2. Remove `TestWriteCSV` below.

The various tests are described through the test method names anyway (as in `test_write_json()` vs. `test_write_csv()` etc.).